code/ui.py

```diff
@@ -39,7 +39,7 @@
 # import filedialog module
 
 
-def process_(file):  # , progress_callback, finish_callback):
+def process_(file,):  # , progress_callback, finish_callback):
```

Comment:
Is there supposed to be a comma after file here?

Reply:
It shouldn't be, but it doesn't matter as Python function syntax will allow it. 
@varundeepakgudhe can fix it.